pkg/cmd/condition/delete.go

```diff
@@ -20,8 +20,8 @@ import (
 	"github.com/spf13/cobra"
 	"github.com/tektoncd/cli/pkg/cli"
 	"github.com/tektoncd/cli/pkg/deleter"
+	namespacecheck "github.com/tektoncd/cli/pkg/namespace"
```

Comment:
Any reason to use `namespacecheck` alias ? I tend to prefer `namespace.Exists(p)` as it's straightforward : "does the namespace exists ?" :upside_down_face:

Reply:
My main thought with this was to avoid confusion if namespace is already defined as a variable somewhere.